Tests/UnitTests/EdgeNetworkServiceTests.swift

```diff
@@ -73,20 +80,26 @@ class EdgeNetworkServiceTests: XCTestCase {
                                  streaming: nil,
                                  responseCallback: mockResponseCallback,
                                  completion: { success, retryInterval  in
-                                    // verify
-                                    XCTAssertTrue(success)
-                                    XCTAssertNil(retryInterval)
-                                    XCTAssertTrue(self.mockNetworking.connectAsyncCalled)
-                                    XCTAssertEqual(testHeaders.count + self.defaultNetworkingHeaders.count, self.mockNetworking.connectAsyncCalledWithNetworkRequest?.httpHeaders.count)
-                                    for header in testHeaders {
-                                        XCTAssertNotNil(self.mockNetworking.connectAsyncCalledWithNetworkRequest?.httpHeaders[header.key])
-                                    }
-
-                                    for header in self.defaultNetworkingHeaders {
-                                        XCTAssertNotNil(self.mockNetworking.connectAsyncCalledWithNetworkRequest?.httpHeaders[header])
-                                    }
-                                    expectation.fulfill()
-                                 })
+            // verify
+            let networkRequests = self.mockNetworkService.getNetworkRequestsWith(url: self.url, httpMethod: .post)
+            guard let networkRequest = networkRequests.first else {
+                XCTFail("Unable to find valid network request.")
+                return
+            }
+
+            XCTAssertTrue(success)
+            XCTAssertNil(retryInterval)
+            XCTAssertEqual(1, networkRequests.count)
+            XCTAssertEqual(testHeaders.count + self.defaultNetworkingHeaders.count, networkRequest.httpHeaders.count)
+            for header in testHeaders {
+                XCTAssertNotNil(networkRequest.httpHeaders[header.key])
+            }
+
+            for header in self.defaultNetworkingHeaders {
+                XCTAssertNotNil(networkRequest.httpHeaders[header])
+            }
+            expectation.fulfill()
+        })
```

Comment:
Can you double check the indentation here? You may need to convert tabs to spaces if you are using them.